src/components/block-tunes/block-tune-delete.ts

```diff
@@ -63,23 +64,15 @@ export default class DeleteTune implements BlockTune {
   }
 
   /**
-   * Create "Delete" button and add click event listener
-   *
-   * @returns {HTMLElement}
+   * Tune's appearance in block settings menu
    */
-  public render(): HTMLElement {
-    this.nodes.button = $.make('div', [this.CSS.button, this.CSS.buttonDelete], {});
-    this.nodes.button.appendChild($.svg('cross', 12, 12));
-    this.api.listeners.on(this.nodes.button, 'click', (event: MouseEvent) => this.handleClick(event), false);
-
-    /**
-     * Enable tooltip module
-     */
-    this.api.tooltip.onHover(this.nodes.button, this.api.i18n.t('Delete'), {
-      hidingDelay: 300,
-    });
-
-    return this.nodes.button;
+  public get blockSettings(): PopoverItem {
```

Comment:
Dont you think, that we could use the existed `render` method? Just extend possible return types

Reply:
i think method `render` is not suitable for such role. There's no rendering, we just need to return a popover entry configuration

Reply:
Agree. So we had to add the new API method and explain it to the world. Ok.

But I don't like the `blockTune.blockSettins` property — it makes no sense. Maybe `toolbox`? Just like at the Tools API